src/elm-json.ts

```diff
@@ -0,0 +1,188 @@
+      args: [
+        {
+          name: "PACKAGE",
+          description:
+            "Package to install, e.g. elm/core or elm/core@1.0.2 or elm/core@1",
+          debounce: true,
+          generators: packageList,
+        },
+        {
+          name: "-- INPUT",
+          description: "The elm.json file to upgrade [default: elm.json]",
+        },
+      ],
+    },
+    {
+      name: "new",
+      description: "Create a new elm.json file",
+      options: [
+        {
+          name: ["--help", "-h"],
+          description: "Prints help information",
+        },
+        {
+          name: ["--version", "-V"],
+          description: "Prints version information",
+        },
+      ],
+    },
+    {
+      name: "tree",
+      description: "List entire dependency graph as a tree",
+      options: [
+        {
+          name: ["--help", "-h"],
+          description: "Prints help information",
+        },
+        {
+          name: "--test",
+          description: "Promote test-dependencies to top-level dependencies",
+        },
+        {
+          name: ["--version", "-V"],
+          description: "Prints version information",
+        },
+      ],
+      args: [
+        {
+          name: "PACKAGE",
+          description:
+            "Limit output to show path to some (indirect) dependency",
+          debounce: true,
+          generators: packageList,
+        },
+        {
+          name: "-- INPUT",
+          description: "The elm.json file to solve [default: elm.json]",
+        },
+      ],
+    },
+    {
+      name: "uninstall",
+      description: "Uninstall a package",
+      options: [
+        {
+          name: ["--help", "-h"],
+          description: "Prints help information",
+        },
+        {
+          name: ["--version", "-V"],
+          description: "Prints version information",
+        },
+        {
+          name: "--yes",
+          description: 'Answer "yes" to all questions',
+        },
+      ],
+      args: [
+        {
+          name: "PACKAGE",
+          description: "Package to uninstall, e.g. elm/html",
+          debounce: true,
+          generators: packageList,
+        },
+        {
+          name: "-- INPUT",
+          description: "The elm.json file to upgrade [default: elm.json]",
+        },
+      ],
+    },
+    {
+      name: "upgrade",
+      description: "Bring your dependencies up to date",
+      options: [
+        {
+          name: ["--help", "-h"],
+          description: "Prints help information",
+        },
+        {
+          name: "--unsafe",
+          description: "Allow major versions bumps",
+        },
+        {
+          name: ["--version", "-V"],
+          description: "Prints version information",
+        },
+        {
+          name: "--yes",
+          description: 'Answer "yes" to all questions',
+        },
+      ],
+      args: {
+        name: "INPUT",
+        description: "The elm.json file to upgrade [default: elm.json]",
```

Comment:
https://fig.io/docs/reference/generator/prebuilt-generators#filepathsoptions
```suggestion
        description: "The elm.json file to upgrade [default: elm.json]",
        generators: [filepaths({ extensions: ["json"] })]
```